code-challenges.test.js

```diff
@@ -15,42 +15,200 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
-
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
+// Create a test wtih describe that takes in fibSequence as a function
+// It takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+// expects two results
+
+describe("fibSequence", () => {
+    it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.", () => {
+        const fibLength1 = 6
+        // Expected output: [1, 1, 2, 3, 5, 8]
+        const fibLength2 = 10
+        // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+        expect(fibSequence(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+        expect(fibSequence(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+}) 
+
+// Red
+// FAIL  ./code-challenges.test.js
+// fibSequence
+//   ✕ takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence. (1 ms)
+// ● fibSequence › takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+//   ReferenceError: fibSequence is not defined
 
 // b) Create the function that makes the test pass.
 
+// Create a function named fibSequence that takes in length as a parameter
+// Create variables, result will be our array starting with our two starting nums, num1 with be the first number in our array, num2 will be the second number in our array, next will be the next number.
+// for loop with i set to 2(since our result starts with 2 values and we don't need to do anything to either) and as long as i is less than our length param we'll do a lot of confusing reassigning
+// next will be the sum of num1 and num2
+// num1 will become num2
+// num2 will become next 
+// we want to push next into our result array
+// we also want to update count so our iteration continues until our conditon is met
+// pass test
+
+const fibSequence = (length) => {
+    let result = [1, 1],
+    num1 = result[0],
+    num2 = result[1],
+    next
+
+    for(let i = 2; i < length; i++) {
+        next = num1 + num2
+        num1 = num2
+        num2 = next
+        result.push(next)
+    }
+    return result
+}
+// This question was extremely difficult for me for some reason, but giving up is not an option for me! I had to look at some outside examples and the formula for the Fibonacci sequence. I also learned I can create multiple variables with one variable declaration and some commas which is cool. 
```

Comment:
This is some solid logic here. You are utilizing some decent techniques I have see a few times.  If you track with this logic then more power to you. I like using other approaches to increase readability for myself mostly.